### ISSUE TITLE
Double-controller detected in Rocket league

### ISSUE THREAD

**Issue by GerardoGR** [lutris/wine#106]
While playing rocket-league (installed via the Epic Games store) with `Lutris v0.5.8.3` and `Lutris wine 6.14 fshack`, If I switch the `Autoconfigure joypads` toggle on, the game detects 2 controllers, If I press the default reverse button in game a new player joins which is a clone of my controller (it follows my movement). 

For it to work I ended up adding these 2 flags to the `user.reg` in the game prefix and switching off the `Autoconfigure joypads` toggle:

```
[Software\\Wine\\DirectInput\\Joysticks] 1628263263
#time=1d78ad6ab54ffb6
"Pro Controller (event)"="disabled"
"Pro Controller (js)"="disabled"
```
Which seem to fix this. Although I'm not exactly sure why, I would've expected to not have any controller at all.

**Comment by tannisroot** [comment]
Should be fixed with 6.21

**Comment by GerardoGR** [comment]
@tannisroot Indeed!
I just tested yesterday with `Lutris Wine 6.21-3` and I have no problems anymore. Thanks!
